src/e3/anod/spec.py

```diff
@@ -245,9 +245,10 @@ def __init__(
         # Create the QualifiersManager.
         # Skip if the name generator is disabled.
         if self.enable_name_generator:
-            self.qualifiers_manager = QualifiersManager(self)
-            self.declare_qualifiers_and_components(self.qualifiers_manager)
-            self.qualifiers_manager.parse(self.parsed_qualifier)
+            if self.kind != "source":
```

Comment:
I know it is not the point of this MR, but knowing the number of references to the `"source"` string, I guess we could define an `is_source` field (which would refer to the `SOURCE_PRIMITIVE` value ?
As it is not the place, I will create a new Issue in github :) Sorry for the noise :)